extensions/standard-processors/processors/DefragmentText.cpp

```diff
@@ -173,7 +173,7 @@ bool DefragmentText::splitFlowFileAtLastPattern(core::ProcessSession& session,
   }
   const auto split_position = gsl::narrow<int64_t>(getSplitPosition(last_regex_match, pattern_location_));
   if (split_position != 0) {
-    split_before_last_pattern = session.clone(*original_flow_file, 0, split_position);
+    split_before_last_pattern = session.clone(*original_flow_file, 0, gsl::narrow<int64_t>(split_position));
```

Comment:
Is this a cast to the very same type?

Reply:
Good point, it seems that in the property refactor the gsl::narrow was added on the above line, so this is not needed anymore, removed in https://github.com/apache/nifi-minifi-cpp/pull/1918/commits/3549d73183c9d531a4fbd449bc6415997b83eb82